gtsfm/common/image.py

```diff
@@ -48,27 +50,57 @@ def get_intrinsics_from_exif(self) -> Optional[Cal3Bundler]:
             intrinsics matrix (3x3).
         """
 
```

Comment:
not sure if get_intrinsics_from_exif is a good name anymore, because it gets intrinsics even if not from exif. 

Anyway, could document that here? "Computes the focal length from EXIF data, but returns a default based on image dimensions if EXIF not found".

Reply:
Done